version.properties

```diff
@@ -4,8 +4,8 @@
 wordpress.versionName=17.6-rc-4
 wordpress.versionCode=1065
 
-wordpress.alpha.versionName=alpha-301
-wordpress.alpha.versionCode=1066
+wordpress.zalpha.versionName=alpha-301
+wordpress.zalpha.versionCode=1066
```

Comment:
This relies on the fact that the pending changes in https://github.com/wordpress-mobile/release-toolkit/pull/283 that this PR points to (see `fastlane/PluginFile` and `Gemfile.lock`) now know that the name of the flavor for alpha builds is `zalpha`, see [here](https://github.com/wordpress-mobile/release-toolkit/pull/283/files#diff-6f51a72fba2128ef36317c1bf3b66ea1f8c47b7f38f2306f9c4b9b34bdfd618aR100-R108)

We intend to improve that to clean up the code in release toolkit later (especially not hardcode `zalpha` in release toolkit nor rely on the `HAS_ALPHA_VERSION` env var, but instead pass the alpha flavor name as parameter of each action) but this clean up of the toolkit will be for another day – tracked in https://github.com/wordpress-mobile/release-toolkit/issues/277